lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -4,17 +4,15 @@
 
 import ddt
 from django.conf import settings
+from milestones.tests.utils import MilestonesTestCaseMixin
 from nose.plugins.attrib import attr
 
-from xmodule.html_module import CourseInfoModule
+from openedx.features.course_experience.views.course_updates import CourseUpdatesFragmentView
```

Comment:
Nit: it seems confusing to have the mobile API make a call out to another Django app's view to get these states. If you have to do another round, I'd move these constants somewhere more central, or maybe just leave them in html_module until there's a better place to put them.